### DISCUSSION TITLE
Start/Stop/check status of recording with intents?

### DISCUSSION THREAD

**Discussion by DDzwiedziu** [FossifyOrg/General-Discussion#650] [Q&A] [answered]
Is there a way to start, stop, check the status of recording with intents?

I wanted to use KeyMapper and a Bluetooth shutter remote to improvise a dictaphone.

**Comment by naveensingh** [comment, marked as answer]
https://github.com/FossifyOrg/Voice-Recorder/issues/98 will cover this.
